CMakeLists.txt

```diff
@@ -17,23 +17,36 @@ cmake_minimum_required(VERSION 3.12)
 # Command line argument takes precedence over environment variable.
 # Loading this before project definition to allow setting the compiler.
 if (MACHINE_CFG)
-  if(EXISTS "${MACHINE_CFG}")
-    include(${MACHINE_CFG})
-  else()
-    message(FATAL_ERROR "Given machine configuration at "
-      "${MACHINE_CFG} not found.")
-  endif()
+  set(USE_MACHINE_CFG ${MACHINE_CFG})
 elseif (DEFINED ENV{MACHINE_CFG})
-  if(EXISTS "$ENV{MACHINE_CFG}")
-    include($ENV{MACHINE_CFG})
-  else()
-    message(FATAL_ERROR "Given machine configuration from environment variable "
-      "MACHINE_CFG at $ENV{MACHINE_CFG} not found.")
+  set(USE_MACHINE_CFG $ENV{MACHINE_CFG})
+endif()
+
+if (USE_MACHINE_CFG)
+  # The `:PAR_ROOT:` string allows the devleper to specify a relative path to
```

Comment:
Nice improvement!

Why use `:` to delineate`:PAR_ROOT:` instead of something else? Just curious. The more common syntax I've seen is `@`

Reply:
Yeah, I can do that instead - it was just the first thing that came to mind.

Reply:
I think I would prefer `@`, but not a big deal either way.

Reply:
Fixed.